gantry/tests/defs/db.py

```diff
@@ -0,0 +1,5 @@
+# flake8: noqa
+# fmt: off
+
+# valid input into insert_node
+NODE_INSERT_DICT = {"uuid": "ec253b04-b1dc-f08b-acac-e23df83b3602", "hostname": "ip-192-168-86-107.ec2.internal", "cores": 24.0, "mem": 196608000000.0, "arch": "amd64", "os": "linux", "instance_type": "i3en.6xlarge"}
```

Comment:
@alecbcs I'm thinking of keeping these definition files (don't have better name lol) as python instead of JSON so we can have comments that explain where these baseline values come from. Not sure if you had a better solution for this setup in mind